apps/docs/src/guide/npm-create-fuels/deploying-a-dapp-to-testnet.md

```diff
@@ -0,0 +1,84 @@
+# Deploying a dApp to Testnet
+
+In this guide, we will deploy a full-stack dApp bootstrapped with `npm create fuels` to the Fuel beta-5 testnet.
+
+> Make sure you have already bootstrapped a dApp using `npm create fuels`. If you haven't, please follow [this guide](./index.md).
+
+There are mainly two steps to get our dApp live on the testnet:
+
+1. Deploying the Contract to the Testnet
+2. Deploying the Frontend to the Cloud
+
+## Deploying the Contract
+
+We recommend using [`forc`](https://docs.fuel.network/docs/forc/) to deploy your contracts to the testnet.
```

Comment:
@Dhaiwat10 As we spoke, I think you could move the `fuelup` stuff to the beginning, as an installation pre-requisite like the [Rust SDK](https://docs.fuel.network/docs/fuels-rs/getting-started/#installation-guide) does.

We can do the rest in subsequent PRs:

 1. Remove `deploy` command
    - Good to have a programmatic `deploy` example in `fuels.config.ts`
 1. Un-document built-in binaries _(keep them for internal purpose only)_
 1. Remove mention about `bun` and post-install scripts

Reply:
- https://github.com/FuelLabs/fuels-ts/issues/2083
 - https://github.com/FuelLabs/fuels-ts/issues/2084


cc @petertonysmith94